patches/@langchain+community+0.3.30.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@langchain/community/dist/experimental/llms/chrome_ai.js b/node_modules/@langchain/community/dist/experimental/llms/chrome_ai.js
```

Comment:
This (and the `patch-package` / `postinstall-postinstall` dependencies) can be removed later after updating to a newer version of `@langchain/community` what would include https://github.com/langchain-ai/langchainjs/pull/7700.